x-pack/legacy/plugins/fleet/dev_docs/fleet_agents_interactions_detailed.md

```diff
@@ -0,0 +1,46 @@
+# Fleet <-> Agent Interactions
+
+## Agent enrollment and checkin
+
+Fleet workflow:
+
+- an agent enroll to fleet using an enrollmentAPiKey
```

Comment:
Are there docs elsewhere for how this "enrollment" API Key and saved-object are created?

Reply:
Not yet, but I should create one 👍